Source/DfBAdminToolkit/Presenter/ProvisioningPresenter.cs

```diff
@@ -255,7 +255,9 @@ private string ProvisionRoles(IProvisioningModel model, IMainPresenter presenter
                     }
                     else
                     {
-                        errorMessage = ErrorMessages.FAILED_TO_ADD_MEMBER;
+                        item.ProvisionStatus = response.Message;
+                        presenter.UpdateProgressInfo(response.Message);
```

Comment:
No it writes in new ProvisionStatus column..one for each user. It still flashes in that progress area..but you see overall results where you should